nhais-adaptor/adaptor/outgoing/tests/test_interchange_adaptor.py

```diff
@@ -0,0 +1,69 @@
+import unittest
+from testfixtures import compare
+from adaptor.outgoing.interchange_adaptor import InterchangeAdaptor
+from adaptor.outgoing.fhir_helpers.operation_definition import OperationDefinitionHelper as odh
+from adaptor.outgoing.fhir_helpers.tests.fixtures import create_simple_patient, create_simple_practitioner
+from adaptor.outgoing.fhir_helpers.constants import ParameterName, ResourceType, OperationName
+
+
+class InterchangeAdaptorTest(unittest.TestCase):
+    """
+    Test the conversation of fhir to an edifact interchange
+    """
+
+    def test_create_interchange(self):
+        """
+        Test the function to create an edifact interchange
+        """
+        with self.subTest("When the operation is for a Birth Registration"):

```

Comment:
Again, I'm not sure of the value of having a single subtest. Is the goal to provide a friendly description for the test? Couldn't you do that with a message when you make an assertion?